src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/SubjectAccessRequestService.kt

```diff
@@ -34,36 +34,36 @@ class SubjectAccessRequestService(
 
 ) : HmppsPrisonSubjectAccessRequestService {
   override fun getPrisonContentFor(
```

Comment:
Fix number 4 🙈 This was just a warning in the code, presumably the implementation has changed from prisonNumber to prn.